### ISSUE TITLE
Failed to launch debuggee VM.

### ISSUE THREAD

**Issue by jintomenachery** [microsoft/vscode-java-test#843]
Adding `"files.encoding": "shiftjis",`  in settings.json giving below error while running/debug a test from VSCode window. It works fine when I `mvn test`. 

```
[Error - 10:15:43 AM] Oct 20, 2019, 10:15:43 AM [error response][launch]: Failed to launch debuggee VM. 'encoding' options in the launch configuration is not recognized.
Failed to launch debuggee VM. 'encoding' options in the launch configuration is not recognized.
com.microsoft.java.debug.core.DebugException: Failed to launch debuggee VM. 'encoding' options in the launch configuration is not recognized.
	at com.microsoft.java.debug.core.adapter.AdapterUtils.createCompletionException(AdapterUtils.java:257)
	at com.microsoft.java.debug.core.adapter.handler.LaunchRequestHandler.handleLaunchCommand(LaunchRequestHandler.java:96)
	at com.microsoft.java.debug.core.adapter.handler.LaunchRequestHandler.handle(LaunchRequestHandler.java:80)
	at com.microsoft.java.debug.core.adapter.DebugAdapter.lambda$dispatchRequest$0(DebugAdapter.java:85)
	at java.base/java.util.concurrent.CompletableFuture.uniComposeStage(CompletableFuture.java:1106)
	at java.base/java.util.concurrent.CompletableFuture.thenCompose(CompletableFuture.java:2235)
	at com.microsoft.java.debug.core.adapter.DebugAdapter.dispatchRequest(DebugAdapter.java:84)
	at com.microsoft.java.debug.core.adapter.ProtocolServer.dispatchRequest(ProtocolServer.java:118)
	at com.microsoft.java.debug.core.protocol.AbstractProtocolServer.lambda$new$0(AbstractProtocolServer.java:78)
	at io.reactivex.internal.observers.LambdaObserver.onNext(LambdaObserver.java:60)
	at io.reactivex.internal.operators.observable.ObservableObserveOn$ObserveOnObserver.drainNormal(ObservableObserveOn.java:200)
	at io.reactivex.internal.operators.observable.ObservableObserveOn$ObserveOnObserver.run(ObservableObserveOn.java:252)
	at io.reactivex.internal.schedulers.ScheduledRunnable.run(ScheduledRunnable.java:61)
	at io.reactivex.internal.schedulers.ScheduledRunnable.call(ScheduledRunnable.java:52)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)

```

**Comment by jdneo** [comment]
@jintomenachery In the `Java Test Runner` output pannel, you should be able to see the launch configuration for the execution. (If the `java.test.log.level` is set to `verbose`)

Would you mind to take a look at the encoding field, is that also `shiftjis`?

**Comment by jdneo** [comment]
@testforstephen Do you have any ideas?

**Comment by testforstephen** [comment]
Apparently `shiftjis` is not a valid encoding charset. 
@jintomenachery could you share your use case to use `shiftjis` as the file encoding?

**Comment by jintomenachery** [comment]
To port an existing project to new maven environment, I had to move lot of existing java files whose file encoding was shiftjs. Without `"files.encoding": "shiftjis"` configuration, these java files were getting opened in utf-8 and when I saved those files after some modification, i was losing many japanese characters from the file. To force load/save these files(without any loss) I had used shiftjs, in files.encoding config.

I found one workaround to the debugger,

I set `"files.autoGuessEncoding": true`, then am not seeing data loss(during file save) and my debugger is working without any issues.
